language-server/src/main/kotlin/tools/samt/ls/SamtDeclarationLookup.kt

```diff
@@ -56,7 +56,7 @@ class SamtDeclarationLookup private constructor() : SamtSemanticLookup<Location,
     }
 
     companion object {
-        fun analyze(fileNode: FileNode, samtPackage: Package) =
-            SamtDeclarationLookup().also { it.analyze(fileNode, samtPackage) }
+        fun analyze(fileNode: FileNode, samtPackage: Package, userMetadata: UserMetadata) =
```

Comment:
We could rename the parameter to `filePackage` to make sure the reader knows it's related to the `FileNode` and not the root package